qa/L0_backend_python/bls/test.sh

```diff
@@ -36,7 +36,7 @@ rm -fr *.log ./models *.txt
 # FIXME: [DLIS-5970] Until Windows supports GPU tensors, only test CPU
 if [[ ${TEST_WINDOWS} == 0 ]]; then
     pip3 uninstall -y torch
-    pip3 install torch==1.13.0+cu117 -f https://download.pytorch.org/whl/torch_stable.html
+    pip3 install torch==2.2.0+cu121 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
It looks windows related changes may be require classifier: 
```
$ curl https://download.pytorch.org/whl/torch_stable.html | cut -d '"' -f 2 | grep /torch-2 | grep "win" | grep 121
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  626k  100  626k    0     0  1438k      0 --:--:-- --:--:-- --:--:-- 1437k
cu121/torch-2.1.0%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.1.0%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.1.0%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.1.0%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp312-cp312-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp312-cp312-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp39-cp39-win_amd64.whl
```

Reply:
This one is not for Windows I believe